queue_services/entity-emailer/src/entity_emailer/email_templates/NR-REFUND.html

```diff
@@ -1,66 +1,17 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <base href="/">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="referrer" content="origin-when-cross-origin"/>
-    <meta name="author" content="BC Registries and Online Services">
-    <title>Name Request notification from the Business Registry</title>
-    [[style.html]]
-  </head>
+This is confirmation that your name request {{ nr_number }} has been cancelled and will not be reviewed by staff.
 
-  <body>
-    <table class="body-table" role="presentation">
-      <tr>
-        <td>
-          [[header.html]]
+**If you paid by credit card or a BC OnLine account:**
+A refund of **${{ refund_value }} CAD** (total payment value including service fee) will be applied to your original payment method. Refunds are usually processed within 10 to 14 business days.
```

Comment:
Is this a properly formatted value, eg, "$101.50" not "$101.5" ?

Reply:
I don't see there have any formatting function for text format.

Reply:
We can test it later. Not sure how it looks like. It depends on source of data.